.github/workflows/tests.yaml

```diff
@@ -39,10 +39,11 @@ jobs:
       - run: echo "setting versions"
     outputs:
       # JSON encoded array of k8s versions.
-      K8S_VERSIONS: '["1.32.2", "1.31.6", "1.30.10", "1.29.14", "1.28.15"]'
-      VAULT_N: "1.18.5"
-      VAULT_N_1: "1.17.6"
-      VAULT_N_2: "1.16.3"
+      K8S_VERSIONS: '["1.33.1", "1.32.5", "1.31.9", "1.30.13", "1.29.14"]'
```

Comment:
Looking at the k8s releases [page](https://kubernetes.io/releases/), it seems like there's newer versions of each of these as there was a release on 6/17/25. Should we be using those instead?

Reply:
Since we're using kind for the CI tests, we're limited to the kindest/node image versions: https://hub.docker.com/r/kindest/node/tags